frontend/components/Header/Nav/Links/SupportTheGuardian.tsx

```diff
@@ -3,6 +3,8 @@ import { cx, css } from 'react-emotion';
 
 import { serif } from '@guardian/pasteup/fonts';
 import { palette } from '@guardian/pasteup/palette';
+import { getCookie } from '../../../../lib/cookie';
+import { AsyncClientComponent } from '../../../lib/AsyncClientComponent';
```

Comment:
Not one for this PR but it feels like it's time to introduce some kind of module resolution for the `lib` directory so we don't have to use/maintain these relative paths!